backend/routes/user.js

```diff
@@ -44,12 +45,22 @@ router.post("/signup", (req, res) => {
         user
           .save()
           .then((user) => {
-            transporter.sendMail({
-              to: user.email,
-              from: "praduman03k@gmail.com",
-              subject: "Thankypu for signing up to Versely",
-              html: "<h1>Welcome to Versely</h1>",
-            });
+            transporter.sendMail(
+              {
+                to: user.email,
+                from: "pradumandumy12399@gmail.com",
+                subject: "Thankyou for signing up to Versely",
+                html: `<img src="http://res.cloudinary.com/dccplpniz/image/upload/v1678773937/y012gjuhmboj5y9mie0i.png" alt="logo">
```

Comment:
Avoid adding links directly in src, prefer storing it in a variable and add variable in src.